img1_topic_name: "/mvsua_cam/image_raw2"
img2_topic_name: "/mvsua_cam/image_raw3"
img3_topic_name: "/mvsua_cam/image_raw4"
img4_topic_name: "/mvsua_cam/image_raw5"
yolo_model_path: "/YoloModel/RD_final"
cof_threshold: 0.65
nms_area_threshold: 0.5
Heights: [58, 66, 84, 118, 236, 400]
Distances: [5.0, 4.0, 3.0, 2.0, 1.0, 0.5]
imshow: true


In1:
  [ 551.388706, 0.000000, 622.656311,
    0.000000, 551.536582, 190.223500,
    0.000000, 0.000000, 1.000000]
Dis1: [-0.005405, -0.006993, -0.000466, 0.000419, 0.000000]
Ex1: 
    [-0.707107, -0.707107,   0.0000,  -0.0219,
    0.0000,     0.0000,    -1.0000,   0.1300,
    0.707107,  -0.707107,   0.0000,  -0.2454,
    0.0000,     0.0000,     0.0000,   1.0000]


In2:
  [ 551.816816, 0.000000, 569.960497,
    0.000000, 551.736399, 94.749160,
    0.000000, 0.000000, 1.000000]
Dis2: [-0.006035, -0.007303, -0.000733, 0.000238, 0.000000]
Ex2: 
    [0.707107,  -0.707107,   0.0000,  -0.0219,
    0.0000,     0.0000,    -1.0000,   0.1300,
    0.707107,   0.707107,   0.0000,  -0.2454,
    0.0000,     0.0000,     0.0000,   1.0000]


In3:
  [ 549.305354, 0.000000, 640.951863,
    0.000000, 549.178482, 146.327399,
    0.000000, 0.000000, 1.000000]
Dis3: [-0.005144, -0.007835, 0.000271, -0.000686, 0.000000]
Ex3: 
    [0.707107,   0.707107,   0.0000,  -0.0219,
    0.0000,     0.0000,    -1.0000,   0.1300,
    -0.707107,   0.707107,   0.0000,  -0.2454,
    0.0000,     0.0000,     0.0000,   1.0000]


In4:
  [ 550.911025, 0.000000, 629.604206,
    0.000000, 550.845787, 165.814740,
    0.000000, 0.000000, 1.000000]
Dis4: [-0.006915, -0.006944, -0.000547, -0.000318, 0.000000]
Ex4: 
    [-0.707107,  0.707107,   0.0000,  -0.0219,
    0.0000,     0.0000,    -1.0000,   0.1300,
    -0.707107,  -0.707107,   0.0000,  -0.2454,
    0.0000,     0.0000,     0.0000,   1.0000]